index.html

```diff
@@ -40,14 +43,31 @@
             align-items: center;
             width: 100%;
             overflow-y: auto;
+            margin-bottom: 20px;
         }
 
         .card {
+            width: 100%;
             border: none;
             border-radius: 4px;
             padding: 20px;
             margin-bottom: 10px;
             box-shadow: 0 2px 2px rgba(0, 0, 0, 0.14), 0 3px 1px rgba(0, 0, 0, 0.12), 0 1px 5px rgba(0, 0, 0, 0.2);
+            background-color: #fff;
+        }
+
+        .countdown {
+            font-family: 'Roboto Mono', monospace;
+            font-size: 1.5rem;
+            font-weight: 500;
+            color: #424242;
```

Comment:
I suggest using a larger font size, bold styling, and a different font family to improve the appearance of the countdown text. Here's an updated CSS snippet with my proposed changes: